bedrock/firefox/templates/firefox/mobile.html

```diff
@@ -331,6 +331,11 @@ <h3>{{ _('Lightweight') }}</h3>
         <li class="android">
           {{ google_play_button(alt_href=settings.GOOGLE_PLAY_FIREFOX_FOCUS_LINK, anchor_attributes={'data-link-type': 'download', 'data-download-os': 'Android', 'id': 'playStoreLink'}) }}
         </li>
+        {% if l10n_has_tag('focus_apk_link_012018') %}
+        <li class="android-apk">
+            <a href="https://archive.mozilla.org/pub/android/focus/latest/" data-link-type="download" data-download-os="Android">{{ _('Download the APK') }}</a>
```

Comment:
I wonder if we need better analytics attributes here. I'm not sure if we'll be able to tell the difference between a Play store click and an APK click.

Also, extra indent. 😬